tests/foreman/ui_airgun/test_hostcollection.py

```diff
@@ -46,7 +54,14 @@
 
 @fixture(scope='module')
 def module_org():
-    return entities.Organization().create()
+    org = entities.Organization().create()
+    # adding remote_execution_connect_by_ip=Yes at org level
+    entities.Parameter(
```

Comment:
just a question, that is not going affect other cases, right? I mean you change organization that will be used for all tests in the module

Reply:
As per my knowledge, it won't affect. Yeah, I just ran one test apart from my tests to ensure that works well. 

```
/home/okhatavk/Downloads/pycharm-community-2018.1.4/helpers/pycharm/_jb_pytest_runner.py --target test_hostcollection.py::test_positive_end_to_end
Launching py.test with arguments test_hostcollection.py::test_positive_end_to_end in /home/okhatavk/Satellite/robottelo/tests/foreman/ui_airgun

============================= test session starts ==============================
platform linux -- Python 3.4.8, pytest-4.0.2, py-1.7.0, pluggy-0.9.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-1.0.2, cov-2.6.12019-05-14 16:55:56 - conftest - DEBUG - Collected 1 test cases                                                 [100%]

========================== 1 passed in 192.52 seconds ==========================
Process finished with exit code 0
```